docs/conventions.md

```diff
@@ -0,0 +1,32 @@
+# Code Conventions
+
+## Formatting
+
+We completely outsource questions of formatting to
+[Prettier](https://prettier.io/), which frees us to bikeshed about other things
+instead. It is run both as a pre-commit hook and also in CI, deniying any PRs
+which do not adhere to its formatting.\\
```

Comment:
Nit: Is the `\\` at the end of this line a typo?
```suggestion
which do not adhere to its formatting.
```

Reply:
Thanks!